### PR TITLE
Removes more recent link from list element

### LINKED ISSUE #10622: A11y_AzureDataStudioStayHealthy_Welcome_Resources_AI4D: Show more is set as list.
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#NonBenchmark;#DesktopApp;#MAS4.1.2;#MAS1.3.1;#SQL Azure Data Studio;#FTP;#AI4D;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS4.1.2, MAS1.3.1
Tool: Accessibility insights for desktop

**Repro Steps:**
1 . Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Navigate till Resources section on Welcome screen.
4 . Launch accessibility insights for desktop and hover over shore more control.

**Actual:**
On hovering over the control show more it has been set as a list.

**Expected:**
The control should be set as button and announced as button itself.

**User Impact:**
Screen reader users will get wrong control information.

**Attachment link for Reference:**
A11y_AzureDataStudioStayHealthy_Welcome_Resources_AI4D

### PR DESCRIPTION
**Pull request by v-bbrady**
This PR removes the show more button on the welcome page from a list element to a div for accessibility alignment.

This PR fixes #10622

### CODE REVIEW

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 1]
```diff
@@ -148,7 +146,7 @@ export default () => `
 						</div>
 						<p>
 						${escape(localize('welcomePage.gettingStartedBody',
-		"Discover the capabilities offered by Azure Data Studio and learn how to make the most of them."))}
+		"Discover the capabilities offered by Aure Data Studio and learn how to make the most of them."))}
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:149]: typo

**Comment by alanrenmsft** [comment]
i am curious why are there so many commits for this PR. @v-bbrady

**Comment by v-bbrady** [comment]
> 
> 
> i am curious why are there so many commits for this PR. @v-bbrady

@alanrenmsft I believe it's just because i created the branch off an older branch. I can submit the PR with just the one commit. I'll do that now.

**Comment by alanrenmsft** [comment]
> > i am curious why are there so many commits for this PR. @v-bbrady
> 
> @alanrenmsft I believe it's just because i created the branch off an older branch. I can submit the PR with just the one commit. I'll do that now.

yes, please.

**Pull request closed without merging**